cli/management.go

```diff
@@ -0,0 +1,142 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/cheynewallace/tabby"
+	"github.com/mongodb/amboy/rest"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+func management(opts *ServiceOptions) cli.Command {
+	return cli.Command{
+		Name: "manage",
+		Subcommands: []cli.Command{
+			manageListJobs(opts),
+			manageAbortAllJobs(opts),
+			manageCheckJob(opts),
+			manageAbortJob(opts),
+		},
+	}
+}
+
+func manageListJobs(opts *ServiceOptions) cli.Command {
+	return cli.Command{
+		Name:  "list",
+		Flags: opts.managementFlags(),
+		Action: func(c *cli.Context) error {
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+
+			return opts.withManagementClient(ctx, c, func(client *rest.ManagementClient) error {
+				jobs, err := client.ListJobs(ctx)
+				if err != nil {
+					return errors.WithStack(err)
+				}
+
+				t := tabby.New()
+				t.AddHeader("Job ID")
+				for _, j := range jobs {
+					t.AddLine(j)
+				}
+				t.Print()
+
+				return nil
+			})
+
+		},
+	}
+}
+
+func manageAbortAllJobs(opts *ServiceOptions) cli.Command {
+	return cli.Command{
+		Name:  "abort-all",
+		Flags: opts.managementFlags(),
+		Action: func(c *cli.Context) error {
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+
+			return opts.withManagementClient(ctx, c, func(client *rest.ManagementClient) error {
+				return errors.WithStack(client.AbortAllJobs(ctx))
+			})
+
+		},
+	}
+}
+
+func manageCheckJob(opts *ServiceOptions) cli.Command {
+	const jobIDFlagName = "id"
+
+	return cli.Command{
+		Name: "check",
+		Flags: opts.managementFlags(
+			cli.StringSliceFlag{
```

Comment:
Should there be a check to ensure this flag is set?

Reply:
It'll 404, and there's only so much client side validation we can really provide.